jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -490,7 +490,19 @@ public String join(String prefix, String suffix, String delimiter, String[] elem
 	public Class<?> findBootstrapClassOrNull(String name) {
 		return VMAccess.findClassOrNull(name, ClassLoader.bootstrapClassLoader);
 	}
-
+	
+	public boolean isEnableNativeAccess(Module mod) {
+		return mod.implIsEnableNativeAccess();
```

Comment:
These need to be protected by the OPENJDK_METHODHANDLES flag, otherwise this will break builds against the openj9 branch.

Reply:
Fixed.